lib/Optimizer/CodeGen/LowerToQIR.cpp

```diff
@@ -366,7 +366,7 @@ class OneTargetRewrite : public ConvertOpToLLVMPattern<OP> {
     std::string qirQisPrefix(cudaq::opt::QIRQISPrefix);
     std::string instName = instOp->getName().stripDialect().str();
     if (!instOp->template hasTrait<cudaq::Hermitian>() && instOp.getIsAdj())
-      instName += "dg";
+      instName += "__adj";
```

Comment:
If you change it here, then you'll have to update NVQIR, and I'm not sure we want to go into all that. This change may be better accomplished in the base profile pipeline.

Reply:
I'll revert and open a bug.

Reply:
#426